src/main/java/com/exasol/releasedroid/usecases/PlatformName.java

```diff
@@ -11,16 +11,17 @@
     GITHUB, MAVEN;
 
     /**
-     * Create a set of {@link PlatformName}s from Strings.
+     * Create a list of {@link PlatformName}s from Strings.
      *
      * @param platforms one or more platform names as Strings
-     * @return set of members of {@link PlatformName} enum class
+     * @return list of members of {@link PlatformName} enum class
      */
-    public static Set<PlatformName> toSet(final String... platforms) {
-        final Set<PlatformName> platformsList = new HashSet<>();
+    public static List<PlatformName> toList(final String... platforms) {
+        final List<PlatformName> platformsList = new ArrayList<>();
         for (final String platform : platforms) {
             platformsList.add(getPlatformName(platform));
         }
+        platformsList.sort(Comparator.comparingInt(List.of(MAVEN, GITHUB)::indexOf));
```

Comment:
if we are moving this, please anyway extract it to a dedicated method to know why we are doing it.

Reply:
Fixed it